docs/_docs/04_reference/cli-options.md

```diff
@@ -735,6 +736,8 @@ When passed it, sets logging level for terragrunt. All supported levels are:
 
 **CLI Arg**: `--terragrunt-check`<br/>
 **Environment Variable**: `TERRAGRUNT_CHECK` (set to `true`)
+**Commands**:
+- [hclfmt](#hclfmt)
```

Comment:
Is this related to this PR?

Reply:
Yes, and it comes from the format change in https://github.com/gruntwork-io/terragrunt/pull/1893/files/fe655623b8730eeb428e6588fe1f2a48b557767e#diff-0ff83c0cbafb0d058b828ba1777066af4bc6eee4f726cecc0e0351b8f15ee65eR776